lib/fog/compute/storm_on_demand.rb

```diff
@@ -1,18 +1,18 @@
 module Fog
   module Compute
     class StormOnDemand < Fog::Service
-      autoload :Config, 'fog/compute/storm_on_demand/models/config'
-      autoload :Configs, 'fog/compute/storm_on_demand/models/configs'
-      autoload :Image, 'fog/compute/storm_on_demand/models/image'
-      autoload :Images, 'fog/compute/storm_on_demand/models/images'
-      autoload :Notification, 'fog/compute/storm_on_demand/models/notification'
-      autoload :Notifications, 'fog/compute/storm_on_demand/models/notifications'
-      autoload :Product, 'fog/compute/storm_on_demand/models/product'
-      autoload :Products, 'fog/compute/storm_on_demand/models/products'
-      autoload :Server, 'fog/compute/storm_on_demand/models/server'
-      autoload :Servers, 'fog/compute/storm_on_demand/models/servers'
-      autoload :Template, 'fog/compute/storm_on_demand/models/template'
-      autoload :Templates, 'fog/compute/storm_on_demand/models/templates'
+      autoload :Config, File.expand_path('../compute/storm_on_demand/models/config', __FILE__)
+      autoload :Configs, File.expand_path('../compute/storm_on_demand/models/configs', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.